docs/development/plugn-creation.md

```diff
@@ -1,14 +1,14 @@
-# Plugin Creation
+# Plugn Creation
```

Comment:
We probably don't need to change the docs here, the change makes it sounds weird.